src/installer/installer.go

```diff
@@ -622,16 +625,17 @@ func (i *installer) updateReadyMasters(nodes *v1.NodeList, readyMasters *[]strin
 			*readyMasters = append(*readyMasters, node.Name)
 			host, ok := inventoryHostsMap[strings.ToLower(node.Name)]
 			if !ok {
-				log.Warnf("Node %s is not in inventory hosts", node.Name)
-				break
+				return fmt.Errorf("Node %s is not in inventory hosts", node.Name)
```

Comment:
as i said previously i am not sure it is right behavior. I think that ready master is part of the inventory hosts and not return error from here. To say the truth if one of the masters is not in the list there is no many sense to continue installation and will be nice to generate event in cluster regarding it.
@eranco74 what do you think?

Reply:
I think that in case the `node.Name` isn't in the map we will fail on timeout anyway (prior to this change there we will just log this warning till the timeout due to the break in the next line).
So I think it's better to fail early.
@tsorya do you think there is any scenario that we get this error and later we recover?